examples/NationalInstruments/highlights/example-points.js

```diff
@@ -0,0 +1,37 @@
+/*global jQuery, $*/
+/*jshint browser: true*/
+
+$(function () {
+    'use strict';
+    var d1 = [];
+    for (var i = 0; i < 14; i += 0.5) {
+        d1.push([i, Math.sin(i)]);
+    }
+
+    var d2 = [[0, 3], [4, 8], [8, 5], [9, 13], [11, 3]];
+
+    // A null signifies separate line segments
+
+    var d3 = [[0, 12], [7, 12], null, [7, 2.5], [12, 2.5]];
+
+    var d4 = [[0, 0], [1, 1], [2, 2], [3, 3], [4, 4]];
+    $.plot("#placeholder", [ d1, d2, d3 ], {
+        series: {
+            lines: { show: true },
+            points: { show: true },
+            highlights: {
+                show: true,
+                highlightColor: '#00FF00',
+                highlightLines: false,
+                highlightPoints: true,
+                highlightBars: false,
+                lineWidth: 5,
+                selectedIndexes:[[0, 2, 3, 4, 5], [0,1,2], [0,2,3]]
```

Comment:
As discussed, it would be nice to include an option to select by a range.

Reply:
Done